lib/bitvec.h

```diff
@@ -209,13 +213,16 @@ class bitvec {
             for (size_t i = sz; i < size; i++)
                 ptr[i] = 0; } }
     bool clrbit(size_t idx) {
+        assert(static_cast<int>(idx) >= 0);
```

Comment:
Unnecessary as this one can't cause problems (clearing a bit past the end of the bitvec is a noop